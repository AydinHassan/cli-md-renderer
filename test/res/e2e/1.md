Now that there is the Tec-9, a crappy spray gun from South Miami. This gun is advertised as the most popular gun in American crime. 
Do you believe that shit? It actually says that in the little book that comes with it: the most popular gun in American crime. Like they're actually proud of that shit. 

----------------------------------------------------------------------
## Hold on to your butts

Some inline code here`$argv`.

Something super interesting:

```php
var_dump($argv);
```

```sh
$ php program.php 1 2 3
```

In which case the output would be an array looking something like:

```php
array(4) {
  [0] =>
  string(7) "program.php"
  [1] =>
  string(1) "1"
  [2] =>
  string(1) "2"
  [3] =>
  string(1) "3"
}
```

[http://slipsum.com]()

*emphasis*

**strong**