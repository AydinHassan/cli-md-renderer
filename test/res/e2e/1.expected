Now that there is the Tec-9, a crappy spray gun from South Miami. This gun is advertised as the most popular gun in American crime.
Do you believe that shit? It actually says that in the little book that comes with it: the most popular gun in American crime. Like they're actually proud of that shit.

[90m------------------------------[0m

[90m##[0m [36m[1mHold on to your butts[0m[0m

Some inline code here[33m$argv[0m.

Something super interesting:

    [36mvar_dump[0m([34m$argv[0m)[33m;[0m
    [0m[0m
    
    [33m$ php program.php 1 2 3
    [0m
In which case the output would be an array looking something like:

    [33marray[0m([95m4[0m) {
      [[95m0[0m] =>
      [36mstring[0m([95m7[0m) [32m"program.php"[0m
      [[95m1[0m] =>
      [36mstring[0m([95m1[0m) [32m"1"[0m
      [[95m2[0m] =>
      [36mstring[0m([95m1[0m) [32m"2"[0m
      [[95m3[0m] =>
      [36mstring[0m([95m1[0m) [32m"3"[0m
    }
    [0m[0m
    
[94m[1m[4mhttp://slipsum.com[0m[0m[0m

[3memphasis[0m

[1mstrong[0m

